test/libsolidity/smtCheckerTests/inheritance/base_contract_assertion_fail_3.sol

```diff
@@ -0,0 +1,31 @@
+pragma experimental SMTChecker;
+
+contract A {
+	uint x;
+	function f() internal virtual {
+		v();
+		assert(x == 0); // should fail
+		assert(x == 2); // should hold
+	}
+	function v() internal virtual {
+		x = 0;
+	}
+}
+
+contract B is A {
+	function f() internal virtual override {
```

Comment:
Can you make this public? Would be a nice test to see the different counter-example.

Reply:
I think `base_contract_assertion_fail_6.sol` (also added in this PR) has the test you mean, but I can also add this one to check.

Reply:
Added